src/main/java/com/endava/endabank/dto/user/UserDetailsDto.java

```diff
@@ -0,0 +1,19 @@
+package com.endava.endabank.dto.user;
+
+import lombok.Data;
+import org.springframework.security.core.GrantedAuthority;
+
+import java.util.Collection;
+
+@Data
+public class UserDetailsDto {
+    private String email;
+    private String phoneNumber;
+    private String firstName;
+    private boolean isApproved;
+    private Collection<String> authorities;
+
+    public void setAuthorities(Collection<? extends GrantedAuthority> authorities) {
+        this.authorities = authorities.stream().map(GrantedAuthority::getAuthority).toList();
```

Comment:
Hi,

When you have a dto class is not a good practice to have logic like this toLowerCase(), it would be better if you have that part in another utility class or when you need it.

http://www.juntadeandalucia.es/servicios/madeja/sites/default/files/historico/1.3.0/contenido-libro-pautas-38.html

"Un DTO normalmente no provee lógica de negocios o validaciones de ningún tipo. Solo provee acceso a las propiedades del bean."